File: src/test/java/no/nav/pensjon/innsyn/domain/source/HardcodedDagpenger.java
package no.nav.pensjon.innsyn.domain.source;

import no.nav.pensjon.innsyn.domain.Dagpenger;
import no.nav.pensjon.innsyn.source.EntityGetter;

import java.util.List;

public class HardcodedDagpenger implements EntityGetter<Dagpenger> {

    @Override
    public List<Dagpenger> getEntities() {
        return List.of(
                new Dagpenger(
                        123456.78d,
                        123456.78d,
                        123456.78d,
                        123456.78d,
                        2019,
                        "kilde",
                        "status",
                        "type",
                        "rapporttype"));
    }
}

File: src/test/kotlin/no/nav/pensjon/innsyn/domain/source/HardcodedDagpenger.kt
package no.nav.pensjon.innsyn.domain.source

import no.nav.pensjon.innsyn.domain.Dagpenger

class HardcodedDagpenger : EntityGetter<Dagpenger?> {
    override val entities = listOf(
                Dagpenger(
                        123456.78,
                        123456.78,
                        123456.78,
                        123456.78,
                        2019,
                        "kilde",
                        "status",
                        "type",
                        "rapporttype"))
}